base/ltfssaxes.dtx

```diff
@@ -152,32 +154,90 @@
 % \begin{itemize}
 % \item
 %
-%    We make \texttt{m} to reset both weight and width (as this is
+%    We make \texttt{m} reset both weight and width (as this is
 %    how it always worked). To reset just the width \texttt{?m} is
-%    provided and to reset just the weight \texttt{m?}.
+%    provided and to reset just the weight \texttt{m?}\ is provided.
 %
 % \item
 %
-%   We do support ``\texttt{m}\textit{width}'' and
-%    ``\textit{weight}\texttt{m}'', e.g., \texttt{mec} to mean ``go to
+%    We support ``\texttt{m}\meta{width}'' and
+%    ``\meta{weight}\texttt{m}'', e.g., \texttt{mec} to mean ``go to
 %    medium weight and extra-condensed width''. At the end of the
 %    process we automatically drop any leftover \texttt{m} in the
 %    series name (unless it is just a single \texttt{m}).
 %
+% \item
+%
+%    If there is no table entry then the requested series is used
+%    unconditionally. This means that we usually do not need entries
+%    where the second argument (the requested series) and the third argument
+%    (the result series) are identical (unless we want to use the fourth
+%    argument to specify an alternative result series). In particular,
+%    this means:
+%    \begin{itemize}
+%    \item
+%
+%      Any request for \texttt{m} needs no entry, i.e., there are no entries
+%      which have \texttt{m} as the second argument.
+%
+%    \item
+%
+%       Any request to set both weight and width (e.g., \texttt{sbx} or
+%       \texttt{ulc}) needs no entry. For that reason, there are no entries
+%       which have a weight+width as second argument (except for cases
+%       involving \texttt{bx}, see below). In particular, this is also true
+%       for cases involving \texttt{m}, e.g., \texttt{bm} (bold medium width)
+%       which automatically gets reduced to \texttt{b}, or \texttt{mc}
+%      (medium weight condensed) which becomes \texttt{c} as a result.
+%
+%       There is one exception: we do have rules for a request for
+%       \texttt{bx}, because the Computer Modern fonts (or related
+%       families like Latin Modern) need \texttt{bx} as their bold series
+%       default (\cs{bfdefault});
+%       this is since only have a \texttt{b} series for a
```

Comment:
quite likely, but the ; in 196 doesn't make sense to me in that rewrite

Reply:
Yes, this sentence still needs some adjustment: too many reasons for reasons for . . .

Reply:
If you have a proposal, I'm happy to change, though that could go in also after the merge. Anyway, the current sources is this
```
%       There is one exception: we do have rules for a request for
%       \texttt{bx}; this is because the Computer Modern fonts (or related
%       families like Latin Modern) need \texttt{bx} as their bold series
%       default (\cs{bfdefault}),
%       since they only have a \texttt{b} series for a
%       restricted set of font faces. Many other font families have only
%       \texttt{b} but no \texttt{bx}, so we have explicit rules
%       that fall back from \texttt{bx} to \texttt{b}.
```

Reply:
In fact this current version looks OK to me now.